### PR TITLE
docs: Form for bug issue creation

### PR DESCRIPTION
**Pull request by janmedrek**
**Description**

Changes proposed in this pull request:

- Add a GitHub Form for the bug issue creation

You can check the looks [here](https://github.com/janmedrek/lifecycle-manager/blob/bug-template/.github/ISSUE_TEMPLATE/bug-report.yml). Formatting is a bit off though.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      description: Provide a general description of the bug that you would like to report.
+      placeholder: Provide your bug description here.
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce
+      description: Provide a detailed list of steps required to reproduce this issue.
+      placeholder: Provide your list here.
```
Comment (mmitoraj) [inline .github/ISSUE_TEMPLATE/bug-report.yml:25]:
```suggestion
      placeholder: Provide your list of steps here.
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      value: |
+        Kubernetes Version: x.y.z
+        Docker Version: x.y.z
+        Minikube Version: x.y.z
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ### To help us identify and resolve the issue efficiently please consider attaching the following:
```
Comment (mmitoraj) [inline .github/ISSUE_TEMPLATE/bug-report.yml:52]:
```suggestion
        ### To help us identify and resolve the issue efficiently, please consider attaching the following:
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        - Kyma CR of the affected cluster.
+        - Manifest CR of the affected module.
+        - ModuleTemplate CR that you have used.
+        **Any additional input will speed up the work and greatly help us with our efforts.**
+  - type: textarea
+    id: attachments
+    attributes:
+      label: Attachments
+      description: Attach any files, links, code samples, or screenshots that will help us debug the issue.
+      placeholder: Put your attachments in here.
```
Comment (mmitoraj) [inline .github/ISSUE_TEMPLATE/bug-report.yml:62]:
```suggestion
      placeholder: Add your attachments here.
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      options:
+        - Managed
+        - minikube
+    validations:
+      required: true
+  - type: textarea
+    id: environment-info
+    attributes:
+      label: Environment Info
+      description: More information on the environment where the bug was found.
```
Comment (mmitoraj) [inline .github/ISSUE_TEMPLATE/bug-report.yml:42]:
```suggestion
      description: More information on the environment where you found the bug.
```
The less passive and more active voice the better :)

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 5]
```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: Report a bug in the project
+title: "Bug Title"
+labels: ["kind/bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Thank you for your report. Before you submit the issue please:
```
Comment (mmitoraj) [inline .github/ISSUE_TEMPLATE/bug-report.yml:9]:
```suggestion
        ### Thank you for your report. Before you submit the issue, please:
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    validations:
+      required: true
+  - type: dropdown
+    id: env-type
+    attributes:
+      label: Environment Type
+      multiple: false
+      options:
+        - Managed
+        - minikube
```
Comment (lindnerby) [inline .github/ISSUE_TEMPLATE/bug-report.yml:35]: Not sure if Minikube is a widely used env option here. We should definitely add k3d as an option here, same  in line 46 to add k3d/k3s version.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+      required: true
+  - type: textarea
+    id: environment-info
+    attributes:
+      label: Environment Info
+      description: More information on the environment where you found the bug.
+      value: |
+        Kubernetes Version: x.y.z
+        Docker Version: x.y.z
+        Minikube Version: x.y.z
```
Comment (lindnerby) [inline .github/ISSUE_TEMPLATE/bug-report.yml:46]: Other versions of interest (if applicable) could be used Kyma CLI

**Review by lindnerby — CHANGES_REQUESTED** [review]

**Review by mmitoraj — DISMISSED** [review]

**Review by lindnerby — APPROVED** [review]

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**